web/src/i18n/translations/ja.yml

```
@@ -139,17 +139,26 @@ Title: タイトル
+Meta Data: メタデータ
+Version History: バージョン履歴
 Item Information: アイテムの情報
 Updated At: 更新日時
 Updated By: 更新者
 Publish State: 公開状態
-Linked Request: リンクされたリクエスト
+Customized meta data: カスタマイズされたメタデータ
+Replace item: アイテムを置き換える
+Refer to item: アイテムを参照
+current: 現在
+Created by: 作成者
```

YAMLlint (1.35.1)

[error] duplication of key "Created by" in mapping

(key-duplicates)